src/Objects/DeviceProperties.php

```diff
@@ -2,9 +2,11 @@
 
 namespace Seam\Objects;
 
+use stdClass;
+
 use function Seam\filter_out_null_params;
 
-class DeviceProperties
+class DeviceProperties extends stdClass
```

Comment:
As an alternative to extending stdClass we can also use the class attribute `#[\AllowDynamicProperties]` at https://www.php.net/manual/en/class.allowdynamicproperties.php but this approach also works 👍

Reply:
Cc @andrii-balitskyi yep we should explore this approach